bcl2fastq/lib/bcl2fastq_utils.py

```diff
@@ -297,6 +297,15 @@ def construct_command(self):
         """
         raise NotImplementedError("Subclasses should implement this!")
 
+    def delete_output(self):
+        """
+        Delete the output directory if it exists
+        :return: None
+        """
+        if os.path.isdir(self.config.output):
```

Comment:
Perhaps you could do some kind of validation that the passed input path is somewhat sensible at least. From what I can tell from https://github.com/arteria-project/arteria-bcl2fastq/blob/723ac28a3c8a7c88cae53ffe7e547c49319e6e59/bcl2fastq/handlers/bcl2fastq_handlers.py#L122 `self.config.output` is just coming straight and unfiltered from the HTTP request.